### PR TITLE
S4 us23/traduction page

### PR DESCRIPTION
**Pull request by Adri-bis**

### CODE REVIEW

**Review by marcDjango — APPROVED** [review]

File: frontend/src/pages/AboutUs.jsx [hunk 1]
```diff
@@ -39,14 +39,13 @@ function AboutUs() {
...
-          aborde avec une créativité débordante. Quand Adrien prend les rênes
-          d'un projet, on peut être sûr que le résultat sera magique, aussi bien
-          visuellement que techniquement.
+          Adrien, it is the web developer who could be confused with a Wizard.
+          Its ability to transform lines of code into websites breathtaking is
+          art. With an almost mystical elegance, he Juggles HTML, CSS and
+          JavaScript as if they were cards from a tarot deck. Each project is a
+          challenge for him, and he approaches with boundless creativity. When
+          Adrien takes the reins of a project, we can be sure that the result
+          will be magical, as well visually and technically.
```
Comment (ArthurHtbk) [inline frontend/src/pages/AboutUs.jsx]: Cadeau => Adrien is the web developer who could be mistaken for a magician. His ability to transform lines of code into stunning websites is truly an art. With an almost mystical elegance, he juggles HTML, CSS, and JavaScript as if they were the cards of a tarot deck. Each project is a challenge for him, and he approaches them with overflowing creativity. When Adrien takes the reins of a project, you can be sure that the result will be magical, both visually and technically.

File: frontend/src/pages/AboutUs.jsx [hunk 2]
```diff
@@ -70,14 +69,13 @@ function AboutUs() {
...
-          sauvegarde impeccables. Marcelo est le gardien des données, veillant à
-          ce que tout fonctionne parfaitement en coulisses, permettant ainsi aux
-          utilisateurs de profiter d'une expérience web sans faille.
+          Marcelo, this is the genius of the backend. He navigates the basics of
+          data and servers with disconcerting ease. For him, the security and
+          stability are the key words of all development web. He enjoys solving
+          complex problems, whether in optimizing SQL queries or setting up
+          monitoring systems impeccable backup. Marcelo is the data keeper,
+          ensuring ensures that everything works perfectly behind the scenes,
+          allowing users to enjoy a seamless web experience.
```
Comment (ArthurHtbk) [inline frontend/src/pages/AboutUs.jsx]: Cadeau => Marcelo is the genius of data management. He navigates databases and servers with disarming ease. For him, security and stability are the watchwords of any web development. He enjoys solving complex problems, whether it's optimizing SQL queries or implementing flawless backup systems. Marcelo is the guardian of data, ensuring that everything runs seamlessly behind the scenes, allowing users to enjoy a seamless web experience.

File: frontend/src/pages/AboutUs.jsx [hunk 3]
```diff
@@ -101,16 +99,16 @@ function AboutUs() {
...
+          Théo, shaman of the four lands, druid of the old world, great PO
+          supreme, conqueror of the five kingdoms, chosen among the chosen,
+          artisan of web design He pays meticulous attention to details and
+          strives to create elegant and intuitive user interfaces. With a keen
+          sense of aesthetics, he juggles with palettes of colors, typographies
+          and graphic elements to bring life to websites that captivate the eye.
+          Every pixel is carefully placed, and each transition is carefully
+          thought out. Theo knows that beauty and friendliness go hand in hand,
+          and he strives to create web experiences that leave a lasting
+          impression.
```
Comment (ArthurHtbk) [inline frontend/src/pages/AboutUs.jsx:111]: Cadeau => Théo, -shaman of the four lands, druid of the ancient world, grand supreme UX designer, conqueror of the five kingdoms, chosen among the chosen, artisan of web design-, pays meticulous attention to details and strives to create elegant and intuitive user interfaces. With a keen sense of aesthetics, he juggles color palettes, typography, and graphic elements to bring life to websites that captivate the eye. Every pixel is carefully placed, and every transition is thought through meticulously. Théo understands that beauty and user-friendliness go hand in hand, and he works to create web experiences that leave a lasting impression.

File: frontend/src/pages/NutriScore.jsx [hunk 4]
```diff
@@ -6,57 +6,49 @@ function NutriScore() {
...
-          l'étiquette.
+          Nutritional values play an essential role in our understanding the
+          nutritional quality of food products. Nutri-Score, for example, is a
+          nutritional labeling system which uses these values to assign an
+          overall rating to a product food, thus facilitating consumer
+          decision-making. Indeed, the Nutri-Score highlights the nutritional
+          value of a produced in a simple and visual way, going from the letter
+          A (green) for the healthiest products with the letter E (red) for the
+          least healthy. This system helps consumers choose foods more balanced
+          and opt for healthier choices, at a glance the label.
```
Comment (ArthurHtbk) [inline frontend/src/pages/NutriScore.jsx]: Cadeau => Nutritional values play an essential role in our understanding of the nutritional quality of food products. Nutri-Score, for example, is a nutritional labeling system which uses these values to assign an overall rating to a product food, thus facilitating consumer decision-making. Indeed, the Nutri-Score highlights the nutritional value of a produced in a simple and visual way, going from the letter A (green) for the healthiest products with the letter E (red) for the least healthy. This system helps consumers choose foods more balanced and opt for healthier choices, at a glance at the label.

File: frontend/src/pages/NutriScore.jsx [hunk 5]
```diff
@@ -6,57 +6,49 @@ function NutriScore() {
...
-          nutritionnelles peuvent parfois être complexes à interpréter.
+          The Nutri-Score takes into account several key nutritional values,
+          such as calories, saturated fats, sugars, salt, proteins, fibers, as
+          well as percentages of the value daily (% DV) of these elements. By
+          combining this information, the Nutri-Score evaluates the overall
+          nutritional quality of a food, in taking into account beneficial
+          nutrients and elements to limit. This labeling system has become a
+          valuable tool for health-conscious consumers, helping them make
+          choices more informed food in an environment where information
+          Nutritional information can sometimes be complex to interpret.
```
Comment (ArthurHtbk) [inline frontend/src/pages/NutriScore.jsx]: Cadeau => The Nutri-Score takes into account several key nutritional values, such as calories, saturated fats, sugars, salt, proteins, fibers, as well as the percentages of the daily value (% DV) of these elements. By combining this information, the Nutri-Score assesses the overall nutritional quality of a food, considering both beneficial nutrients and elements to limit. This labeling system has become a valuable tool for health-conscious consumers, helping them make more informed dietary choices in an environment where nutritional information can sometimes be complex to interpret.

File: frontend/src/pages/NutriScore.jsx [hunk 6]
```diff
@@ -6,57 +6,49 @@ function NutriScore() {
...
-          informés, favorisant ainsi une alimentation équilibrée et une
-          meilleure qualité de vie.
+          The Nutri-Score is an example of how values nutritional practices are
+          put into practice to promote healthy healthier diet. It helps raise
+          consumer awareness nutritional qualities of food products and
+          encourages manufacturers to improve the nutritional composition of
+          their products. Ultimately, the nutritional values and Nutri-Score are
+          powerful tools to promote health public by helping individuals make
+          better food choices informed, thus promoting a balanced diet and
+          better quality of life.
```
Comment (ArthurHtbk) [inline frontend/src/pages/NutriScore.jsx:51]: Cadeau => The Nutri-Score is an example of how nutritional values are put into practice to promote a healthier diet. It contributes to raising awareness among consumers about the nutritional qualities of food products and encourages manufacturers to improve the nutritional composition of their products. Ultimately, nutritional values and the Nutri-Score are powerful tools for promoting public health by assisting individuals in making more informed dietary choices, thereby promoting a balanced diet and a better quality of life.

File: frontend/src/pages/Tomorrow.jsx [hunk 7]
```diff
@@ -8,17 +8,16 @@ function Tomorrow() {
...
-          respectueuses de l'environnement pour vous inspirer à faire des choix
-          alimentaires plus responsables.
+          In a constantly changing world, it is essential that we think about
+          the way we eat and the impact of our choices food on the future of our
+          planet. Challenges environmental issues are increasingly pressing, and
+          our food plays a key role in preserving the Earth. In this article, we
+          will explore the notion of “Let’s think about Tomorrow” in the context
+          sustainable cooking. We will share delicious recipes and
+          environmentally friendly to inspire you to make choices more
+          responsible food.
```
Comment (ArthurHtbk) [inline frontend/src/pages/Tomorrow.jsx]: Cadeau => In a constantly evolving world, it is essential that we reflect on our eating habits and the impact of our dietary choices on the future of our planet. Environmental issues are becoming increasingly urgent, and our diet plays a key role in preserving the Earth. In this article, we will explore the concept of 'Thinking About Tomorrow' in the context of sustainable cooking. We will share delicious and environmentally friendly recipes to inspire you to make more responsible food choices.

File: frontend/src/pages/Tomorrow.jsx [hunk 8]
```diff
@@ -27,29 +26,26 @@ function Tomorrow() {
...
           <p>
-            Le gaspillage alimentaire est un problème majeur dans le monde
-            d'aujourd'hui. En réduisant le gaspillage dans votre cuisine, vous
-            contribuez à la préservation des ressources naturelles. Une astuce
-            simple consiste à planifier vos repas, à conserver les restes et à
-            être créatif avec les ingrédients que vous avez sous la main.
+            Food waste is a major problem worldwide of today. By reducing waste
+            in your kitchen, you contribute to the preservation of natural
+            resources. A tip simple is to plan your meals, save leftovers and be
+            creative with the ingredients you have on hand.
```
Comment (ArthurHtbk) [inline frontend/src/pages/Tomorrow.jsx]: Cadeau => Food waste is a significant issue in today's world. By minimizing waste in your kitchen, you play a part in conserving natural resources. A simple tip is to plan your meals, store leftovers, and get creative with the ingredients you have on hand.

File: frontend/src/pages/Tomorrow.jsx [hunk 9]
```diff
@@ -62,26 +58,24 @@ function Tomorrow() {
...
           <p>
-            La production de viande a un impact environnemental significatif en
-            termes d'utilisation de ressources et d'émissions de gaz à effet de
-            serre. En incorporant davantage de repas à base de plantes dans
-            votre alimentation, vous réduisez votre empreinte carbone et
-            contribuez à la préservation des écosystèmes.
+            Meat production has a significant environmental impact in terms of
+            resource use and gas emissions tight. By incorporating more
+            plant-based meals into your diet, you reduce your carbon footprint
+            and contribute to the preservation of ecosystems.
```
Comment (ArthurHtbk) [inline frontend/src/pages/Tomorrow.jsx]: Cadeau => The production of meat has a significant environmental impact in terms of resource use and greenhouse gas emissions. By incorporating more plant-based meals into your diet, you reduce your carbon footprint and contribute to the preservation of ecosystems.

File: frontend/src/pages/Tomorrow.jsx [hunk 10]
```diff
@@ -62,26 +58,24 @@ function Tomorrow() {
...
           <p>
-            Les aliments biologiques sont produits sans l'utilisation de
-            pesticides chimiques et d'engrais synthétiques, ce qui les rend plus
-            respectueux de l'environnement. En choisissant des produits
-            biologiques, vous soutenez une agriculture durable et préservez la
-            biodiversité.
+            Organic foods are produced without the use of chemical pesticides
+            and synthetic fertilizers, making them more eco-friendly. When
+            choosing products organic, you support sustainable agriculture and
+            preserve the biodiversity.
```
Comment (ArthurHtbk) [inline frontend/src/pages/Tomorrow.jsx]: Cadeau => Organic foods are produced without the use of chemical pesticides and synthetic fertilizers, making them more environmentally friendly. By choosing organic products, you support sustainable agriculture and preserve biodiversity.

File: frontend/src/pages/Tomorrow.jsx [hunk 11]
```diff
@@ -91,14 +85,13 @@ function Tomorrow() {
...
-          délicieuses sont non seulement bonnes pour vous, mais aussi pour
-          l'environnement. Alors, mettez votre tablier et préparez-vous à
-          savourer un avenir plus vert et plus durable.
+          Thinking about tomorrow means adopting a sustainable approach to our
+          food. By making conscious choices, such as eating locally, reduce food
+          waste, opt for a diet based on plants and favor organic foods, we can
+          contribute to a better future for our planet. These recipes delicious
+          foods are not only good for you, but also for the environment. So, put
+          on your apron and get ready to enjoy a greener and more sustainable
+          future.
```
Comment (ArthurHtbk) [inline frontend/src/pages/Tomorrow.jsx]: Cadeau => Thinking about tomorrow means adopting a sustainable approach to our diet. By making conscious choices such as eating locally, reducing food waste, opting for a plant-based diet, and choosing organic foods, we can contribute to a better future for our planet. These delicious recipes are not only good for you but also for the environment. So, put on your apron and get ready to savor a greener and more sustainable future.

**Review by ArthurHtbk — CHANGES_REQUESTED** [review]
Quelques petites fautes mineures dans les textes en anglais, je me suis permis de vous proposer quelques amendements 🙂

**Review by ArthurHtbk — APPROVED** [review]

**Pull request merged**